ext/standard/tests/network/setcookie_error.phpt

```diff
@@ -10,9 +10,15 @@ ob_start();
 // Unrecognized key and no valid keys
 setcookie('name', 'value', ['unknown_key' => 'only']);
 // Numeric key and no valid keys
-setcookie('name', 'value', [0 => 'numeric_key']);
+setcookie('name2', 'value2', [0 => 'numeric_key']);
 // Unrecognized key
-setcookie('name', 'value', ['path' => '/path/', 'foo' => 'bar']);
+setcookie('name3', 'value3', ['path' => '/path/', 'foo' => 'bar']);
+// Arguments after options array (will not be set)
+setcookie('name4', 'value4', [], "path", "domain.tld", true, true);
+
+var_dump(headers_list());
+
+--EXPECTHEADERS--
 
```

Comment:
When we use a `--EXPECTHEADERS--` empty section, what are we trying to achieve?

Reply:
This is so the test is executed with the CGI binary. With the CLI the headers are ignored and the return of `headers_list()` would be empty.